nutrition_labels/grant_data_processing.py

```diff
@@ -1,6 +1,8 @@
 import pandas as pd
 import re
 from bs4 import BeautifulSoup
+from nutrition_labels.useful_functions import remove_useless_string
+from nutrition_labels.useful_functions import only_text
```

Comment:
you can just do from nutrition_labels.useful_functions import remove_useless_string, only_text if you want